### PR TITLE
Zeenath

### PR DESCRIPTION
**Pull request by ZeenatHaseer**

### CODE REVIEW

**Review by toghrulito — APPROVED** [review]

**Review by JavidKK — APPROVED** [review]
Coll

**Review by toghrulito — APPROVED** [review]

**Review by toghrulito — APPROVED** [review]

**Review by toghrulito — APPROVED** [review]

**Review by Maharram1984 — APPROVED** [review]
Sen unutma kimdi Collins

File: configuration.properties [hunk 1]
```diff
@@ -4,5 +4,7 @@ usernameMarketing=marketing21@cybertekschool.com
 passwordMarketing=UserUser
 usernameHr=hr21@cybertekschool.com
 passwordHr=UserUser
-usernameHelpDesk=helpdesk21@cybertekschool.com
-passwordHelpDesk=UserUser
```
Comment (JavidKK) [inline configuration.properties]: great code

**Review by JavidKK — DISMISSED** [review]
great job

**Review by toghrulito — APPROVED** [review]

**Comment by ZibaMammadzada** [comment]
i am approving!

**Pull request closed without merging**